### DISCUSSION TITLE
这个项目是怎么回事?这么多star,怎么都摆烂2年多了.

### DISCUSSION THREAD

**Discussion by sgf** [shadowsocks/shadowsocks-windows#3473] [General]
这个项目是怎么回事?这么多star,怎么都摆烂2年多了.

**Comment by rcz0315** [comment]
这个开源项目，已经可以满足大部分需求，没有重大变动为什么要持续更新？自定义的需求，fork后再编译不香吗😊

**Reply by robot-dot-win** [reply]
> 这个开源项目，已经可以满足大部分需求，没有重大变动为什么要持续更新？自定义的需求，fork后再编译不香吗😊

现在出现了SIP022 AEAD-2022，一些项目只支持AEAD-2022不支持老协议。而这个项目是windows上最好的GUI，只需要加上AEAD-2022支持就可以了。

**Reply by sgf** [reply]
> 现在出现了SIP022 AEAD-2022，一些项目只支持AEAD-2022不支持老协议。而这个项目是windows上最好的GUI，只需要加上AEAD-2022支持就可以了。

我已经用v2rayN 了，之前用ss协议，服务器被秒封

**Comment by w2jmoe** [comment]
等您来更新维护，这不才显得你技术超群吗！？哈哈，开个玩笑... = w =

**Reply by w2jmoe** [reply]
Ps. 龙猫头像可爱~

**Comment by sgf** [comment]
现在不用ss了，直接用v2rayN 了

**Reply by robot-dot-win** [reply]
> 现在不用ss了，直接用v2rayN 了

ss-windows的GUI非常完美，在系统托普图标上可以直接看到有无流量。我给v2rayN提了个需求，结果人家直接说不会做这个功能。我找人fork一个ss-windows，自己做AEAD-2022。

**Reply by sgf** [reply]
> ss-windows的GUI非常完美，在系统托普图标上可以直接看到有无流量。我给v2rayN提了个需求，结果人家直接说不会做这个功能。我找人fork一个ss-windows，自己做AEAD-2022。

我还提了个需求，做客户端流量控制呢。但也被否决了。算了，毕竟是白嫖，将就用用吧。
